app/README.md

```
@@ -2,17 +2,27 @@
 
 ## Installation
 
-1. Create `./.env` in this directory (see `.env.example`).
-2. Edit `./.env` with an appropriate database file location.
-3. Create `$HOME/.bloom/desktop-config.yaml` (see `desktop-config.yaml.example`).
-4. Edit `$HOME/.bloom/desktop-config.yaml` with the correct database file location and other values.
-5. Run `npm install`.
-6. Run `npm run client:generate` to generate Typescript client for the new schema.
-7. Run `npm run db:deploy` to create the database.
+1. `mamba activate bloom-desktop`
+2. Follow the instructions in (bloom)[https://gitlab.com/salk-tm/bloom/-/tree/main?ref_type=heads] to setup a local supabase instance to interact with, and add some test data.
```

Markdownlint

(bloom)[https://gitlab.com/salk-tm/bloom/-/tree/main?ref_type=heads]
Reversed link syntax

(MD011, no-reversed-links)